code/game/machinery/autolathe.dm

```diff
@@ -216,7 +216,7 @@
 	return ..()
 
 /obj/machinery/autolathe/proc/AfterMaterialInsert(type_inserted, id_inserted, amount_inserted)
-	if(ispath(type_inserted, /obj/item/ore/bluespace_crystal))
+	if(ispath(type_inserted, /obj/item/stack/ore/bluespace_crystal))

```

Comment:
This looks like it could use some more logic here, since you are now inserting a stack, and not just a single piece of ore